src/webextension/widgets/link.js

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import PropTypes from "prop-types";
+import React from "react";
+
+import styles from "./link.css";
+
+export function ExternalLink({onClick, children}) {
```

Comment:
defined it this way because I strongly suspect we'll have internal links in the near future, and thought this would help minimize the changes to consumers of this widget.